components/navbar/navbar.tsx

```diff
@@ -8,17 +8,22 @@ import LogoBook from '../svg/logoBook';
 
 interface iNavbar {
   color: string;
+  value?: string;
+  setValue?: (e: any) => void;
```

Comment:
Avoid using `any`. You can take a look at this article related to react and typescript events
https://fettblog.eu/typescript-react/events/